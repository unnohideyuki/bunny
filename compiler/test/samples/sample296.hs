main = do print $ zipWith3 (\x y z -> x + 2*y + 3*z) [1..5] [5..10] [10..15]
          print $ zip3 [1..5] [9,8..7] [1,1,1,1,1]
          print $ unzip3 [(1,2,3), (2,3,4), (3,4,5)]
