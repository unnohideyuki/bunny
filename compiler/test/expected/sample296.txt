[41,47,53,59,65]
[(1,9,1),(2,8,1),(3,7,1)]
